<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>1108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sqlgull</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/icons/silk/icons/custom/seagull.png</normaloff>:/icons/silk/icons/custom/seagull.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="newFileButton">
          <property name="toolTip">
           <string>New File</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/page_white_add.png</normaloff>:/icons/silk/icons/silk/page_white_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openFileButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Open File</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/folder.png</normaloff>:/icons/silk/icons/silk/folder.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveFileButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Save File</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/disk.png</normaloff>:/icons/silk/icons/silk/disk.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="connectionComboBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Switch Connection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newConnectionButton">
          <property name="toolTip">
           <string>New Connection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/database_add.png</normaloff>:/icons/silk/icons/silk/database_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editConnectionButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Edit Connection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/database_edit.png</normaloff>:/icons/silk/icons/silk/database_edit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteConnectionButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete Connection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/database_delete.png</normaloff>:/icons/silk/icons/silk/database_delete.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearCredentialsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Clear Credentials</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/database_lightning.png</normaloff>:/icons/silk/icons/silk/database_lightning.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshMetadataButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Refresh Metadata</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/database_refresh.png</normaloff>:/icons/silk/icons/silk/database_refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="queryBlockButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Query Block At Cursor</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/resultset_next.png</normaloff>:/icons/silk/icons/silk/resultset_next.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelQueryButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Cancel Query</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/icons.qrc">
            <normaloff>:/icons/silk/icons/silk/stop.png</normaloff>:/icons/silk/icons/silk/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabBarConnections">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1450</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveFileAs"/>
    <addaction name="actionCloseFile"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewConnection"/>
    <addaction name="actionEditConnection"/>
    <addaction name="actionDeleteConnection"/>
    <addaction name="actionClearCredentials"/>
    <addaction name="actionRefreshMetadata"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>&amp;Query</string>
    </property>
    <addaction name="actionQueryBlockAtCursor"/>
    <addaction name="actionCancelQuery"/>
    <addaction name="separator"/>
    <addaction name="actionExportResultsToFile"/>
    <addaction name="actionExportResultsToClipboard"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
   <addaction name="menuQuery"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="databaseObjectDockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="databaseObjectDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="DatabaseObjectTreeView" name="databaseObjectTreeView">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/cross.png</normaloff>:/icons/silk/icons/silk/cross.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/information.png</normaloff>:/icons/silk/icons/silk/information.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/page_white_add.png</normaloff>:/icons/silk/icons/silk/page_white_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/folder.png</normaloff>:/icons/silk/icons/silk/folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/disk.png</normaloff>:/icons/silk/icons/silk/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewConnection">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/database_add.png</normaloff>:/icons/silk/icons/silk/database_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Connection</string>
   </property>
  </action>
  <action name="actionEditConnection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/database_edit.png</normaloff>:/icons/silk/icons/silk/database_edit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit Connection</string>
   </property>
  </action>
  <action name="actionDeleteConnection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/database_delete.png</normaloff>:/icons/silk/icons/silk/database_delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete Connection</string>
   </property>
  </action>
  <action name="actionQueryBlockAtCursor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/resultset_next.png</normaloff>:/icons/silk/icons/silk/resultset_next.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Query Block At Cursor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/page_white_delete.png</normaloff>:/icons/silk/icons/silk/page_white_delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/drive_disk.png</normaloff>:/icons/silk/icons/silk/drive_disk.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;File As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExportResultsToFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/page_white_go.png</normaloff>:/icons/silk/icons/silk/page_white_go.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export Results to File</string>
   </property>
  </action>
  <action name="actionClearCredentials">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/database_lightning.png</normaloff>:/icons/silk/icons/silk/database_lightning.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear Credentials</string>
   </property>
  </action>
  <action name="actionCancelQuery">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/stop.png</normaloff>:/icons/silk/icons/silk/stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cancel Query</string>
   </property>
  </action>
  <action name="actionExportResultsToClipboard">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/page_white_copy.png</normaloff>:/icons/silk/icons/silk/page_white_copy.png</iconset>
   </property>
   <property name="text">
    <string>Export &amp;Results to Clipboard</string>
   </property>
  </action>
  <action name="actionRefreshMetadata">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/silk/icons/silk/database_refresh.png</normaloff>:/icons/silk/icons/silk/database_refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh Metadata</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DatabaseObjectTreeView</class>
   <extends>QTreeView</extends>
   <header>ui/DatabaseObjectTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNewFile</sender>
   <signal>triggered()</signal>
   <receiver>newFileButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewConnection</sender>
   <signal>triggered()</signal>
   <receiver>newConnectionButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditConnection</sender>
   <signal>triggered()</signal>
   <receiver>editConnectionButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeleteConnection</sender>
   <signal>triggered()</signal>
   <receiver>deleteConnectionButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQueryBlockAtCursor</sender>
   <signal>triggered()</signal>
   <receiver>queryBlockButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1101</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveFile</sender>
   <signal>triggered()</signal>
   <receiver>saveFileButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenFile</sender>
   <signal>triggered()</signal>
   <receiver>openFileButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearCredentials</sender>
   <signal>triggered()</signal>
   <receiver>clearCredentialsButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCancelQuery</sender>
   <signal>triggered()</signal>
   <receiver>cancelQueryButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefreshMetadata</sender>
   <signal>triggered()</signal>
   <receiver>refreshMetadataButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>870</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
